action.yml

```diff
@@ -15,12 +15,18 @@ env:
 runs:
   using: 'composite'
   steps:
-    - uses: getsentry/action-setup-venv@3a832a9604b3e1a4202ae559248f26867b467cc7 #v2.1.1
+    - uses: astral-sh/setup-uv@884ad927a57e558e7a70b92f2bccf9198a4be546 # v6
+      with:
+        version: '0.8.2'
+        # we just cache the venv-dir directly in action-setup-venv
+        enable-cache: false
+
+    - uses: getsentry/action-setup-venv@3a832a9604b3e1a4202ae559248f26867b467cc7 # v2.1.1
       id: venv
       with:
         python-version: ${{ inputs.python-version }}
-        cache-dependency-path: ${{ inputs.requirements-file-path }}
-        install-cmd: pip install -r ${{ inputs.requirements-file-path }}
+        cache-dependency-path: ""
+        install-cmd: uv pip install "devservices==${{ inputs.devservices-version }}"
```

Comment:
Does it make sense to `uv add` here?

Reply:
No, uv add calculates a lockfile. The way to install a single ad hoc dependency is to use `uv pip`.